versioned_docs/version-latest/protocol/pulsar.mdx

```diff
@@ -86,60 +97,44 @@ You can change these settings for an existing file system:
 sudo tune2fs -E stride=8,stripe-width=8 /path/to/dev
 ```
 
-:::
-
-
-### Getting A Crypto Wallet
-
-Before running anything you need to have a wallet where you'll receive testnet coins.
-Follow our how-to guides in the [Wallets section](/docs/category/wallets/) of the documentation for steps on how to get your wallet setup.
-
-### Required ports
-
-Currently, a few ports need to be exposed for node to work properly.
-
-If you have a server with no firewall, there is nothing to be done, but otherwise make sure to open the following TCP ports for incoming connections.
-
-- `30333`
-- `30433`
-- `30533`
+</details>
 
-On the desktop side if you have a router in front of your computer, you'll need to forward TCP ports to the machine on which your node is running (how this is done varies from router to router, but there is always a feature like this, refer to [How to Forward Ports](port-forwarding.md) for a more in-depth tutorial).
-If you're connected directly without any router, then again nothing needs to be done in such case.
+### Step 1: Creating A Crypto Wallet
 
+The first thing you will need to start is a wallet to receive your farming rewards.
+Follow our how-to guides in the [Wallets section](/docs/category/wallets/) of the documentation for steps on how to setup your wallet.
 
 
-## Installation
+## Step 2: Installation
 
 :::caution ALPHA SOFTWARE
-Pulsar is in **alpha**.
-Please feel free to file bug reports on our GitHub issues.
+Pulsar is in **alpha** and may contain bugs.
 :::
 <details>
   <summary>
-  Prefer A video? Expand for our installation video
+  Prefer a video? Expand this section to view an installation video
   </summary>
   <div>
   <iframe width="560" height="315" src="https://www.youtube.com/embed/h1-CX7lGl2w" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
   </div>
 </details>
 
 
-### Download Binaries
+### Download Pulsar
 
 
 
 ---
 
-Precompiled versions of the Pulsar is [hosted on GitHub](https://github.com/subspace/pulsar/releases). This is the recommended way to install the application. Please find the appropriate binary for your operating system.
+Precompiled versions of Pulsar are [hosted on GitHub](https://github.com/subspace/pulsar/releases). This is the recommended way to install the application. Please find the appropriate binary for your operating system.
 
 <Tabs groupId="OS">
 
 <TabItem value="windows" label="🖼️ Windows" default>
 
 :::caution
 
-Some older processors/VMs are no longer supported by official releases, but can still be compiled manually if desired
+Some older processors/VMs are no longer supported by official releases, but can still be compiled manually.
```

Comment:
What do you think about making **compiled manually** a link here to the "how to compile manually" section?

Reply:
Great idea -- I personally came across a suggestion to compile manually in the forum and didn't know where to start. I found [this link](https://github.com/subspace/subspace/blob/main/docs/development.md) on the Local Development page and will use that unless you suggest otherwise.